tasks/pipeline.py

```diff
@@ -555,9 +555,7 @@ def is_system_probe(owners, files):
 @task
 def changelog(ctx, new_commit_sha):
     old_commit_sha = ctx.run(
-        "aws ssm get-parameter --region us-east-1 --name "
-        "ci.datadog-agent.gitlab_changelog_commit_sha --with-decryption --query "
-        "\"Parameter.Value\" --out text",
+        "$CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh ci.datadog-agent.gitlab_changelog_commit_sha",
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏
We will not be able to use it locally anymore with this

Reply:
`__file__` in Python is _probably_ the most reliable. Otherwise Git root can do as well.

Reply:
I used `os.getcwd` as I guess if you want to launch locally the task you need to be at the root of the git repo, which is what I need